taniumrest_consts.py

```diff
@@ -31,6 +31,7 @@
     "[current result unavailable]",
     "[results currently unavailable]"
 ]
+TANIUMREST_PARAMETERS_WITHOUT_INPUT = ("com.tanium.components.parameters::SeparatorParameter")
```

Comment:
This is not a tuple but a string and thus this whole thing works by accident

Reply:
Agree

Reply:
So are you going to change this?

Reply:
yes 
appreciate your meticulousness.